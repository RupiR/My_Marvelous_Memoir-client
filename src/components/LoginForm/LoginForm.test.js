import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import LoginForm from './LoginForm';

describe('<NewPostForm />', () => {
    it('Renders without crashing', () => {
        shallow(<LoginForm />);
    });

    it('renders correctly', () => {
        const tree = renderer
            .create(<LoginForm />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});