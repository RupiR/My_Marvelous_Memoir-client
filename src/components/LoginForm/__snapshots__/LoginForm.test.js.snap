// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewPostForm /> renders correctly 1`] = `
<form
  className="LoginForm"
  onSubmit={[Function]}
>
  <div
    role="alert"
  />
  <div
    className="username"
  >
    <label
      htmlFor="LoginForm__username"
    >
      Username
    </label>
    <input
      className="Input "
      id="LoginForm__username"
      name="username"
      required={true}
    />
  </div>
  <div
    className="password"
  >
    <label
      htmlFor="LoginForm__password"
    >
      Password
    </label>
    <input
      className="Input "
      id="LoginForm__password"
      name="password"
      required={true}
      type="password"
    />
  </div>
  <button
    className="Button "
    type="submit"
  >
    Login
  </button>
</form>
`;
